config/settings.yml

```diff
@@ -29,6 +29,7 @@ CAP:
 ## PubMed Auth Config
 PUBMED:
   LOG: log/pubmed.log
+  harvest_enabled: true # this only enables/disabling regular harvests, it will still allow manual entries via pubmed and record augmentation
   FETCH_PATH: /entrez/eutils/efetch.fcgi?db=pubmed&retmode=xml
```

Comment:
Should this be `enabled` to stay consistent with WOS?

Reply:
right now its not consistent though...if you set WOS.harvest to disabled, then you aren't allowing any WOS client connections at all (not just harvesting), but pubmed disabled is only for harvesting...not ideal but a stopgap to kill just Pubmed harvesting while allowing single record augmentation